utils/build/build.js

```diff
@@ -130,7 +132,7 @@ class ProgramStep extends Step {
       },
       cwd: step.cwd,
     });
-    process.on('exit', () => child.kill());
+    disposables.push(() => child.kill());
```

Comment:
Should we remove it at some point, or is it fie to call `child.kill()` on a closed child?

Reply:
From their docs:

> Sending a signal to a child process that has already exited is not an error but may have unforeseen consequences. Specifically, if the process identifier (PID) has been reassigned to another process, the signal will be delivered to that process instead which can have unexpected results.

Addressed.